### DISCUSSION TITLE
Colored oozing potion

### DISCUSSION THREAD

**Discussion by ParkR15MineR** [BedrockTweaks/Files#249] [Ideas]
I think it would be cool if you chose one of the colored slimes, the potion effect should also be that color

![IMG_0215](https://github.com/user-attachments/assets/f65a1600-60bb-4e41-885f-8e9ee73e9bd4)

**Comment by ParkR15MineR** [comment]
For instance, if you chose purple slimes, the potion would look purple and the effect icon would also look purple

**Reply by SquatchHunter** [reply]
that seems like a very good idea, something we can turn into an issue and track / merge :)

**Reply by ParkR15MineR** [reply]
Should I make it an issue or will one of you guys do it?

**Reply by PAPISOP** [reply]
DrAv will do it once he approves this suggestion, he is the only one who decides what packs/changes are going to be made

**Reply by ParkR15MineR** [reply]
I see, I see

**Reply by SquatchHunter** [reply]
also, this would actually be a little bit of work to maintain since if we recolour the oozing effect
we have to recolour the icon used in the UI, particles, potion ect ect
not nearly as large as some other packs we have but a lot more to consider

**Reply by ParkR15MineR** [reply]
True, this idea just came to me when I was in the brewing stand and the potion effects on the side used a green slime block instead of a colored one, (I had purple slimes on)

**Reply by PAPISOP** [reply]
> True, this idea just came to me when I was in the brewing stand and the potion effects on the side used a green slime block instead of a colored one, (I had purple slimes on)

Slime block 3d texture is hard coded to be the vanilla texture, although this might he a missing combination but not too sure

**Reply by ParkR15MineR** [reply]
Fascinating

**Reply by SquatchHunter** [reply]
giving the latest update, progress for "data driven items" is underway and this will be revisited once thats stable
